geoips/interfaces/base.py

```diff
@@ -618,20 +537,102 @@ def __repr__(self):
         """Plugin interface repr method."""
         return f"{self.__class__.__name__}()"
 
-    def _plugin_module_to_obj(self, module, module_call_func="call", obj_attrs={}):
-        """Convert a plugin module into an object.
+    # def _plugin_module_to_obj(self, module, module_call_func="call", obj_attrs={}):
+    #     """Convert a plugin module into an object.
+
+    #     Convert the passed module into an object of type.
+    #     """
+    #     obj = plugin_module_to_obj(
+    #         module=module, module_call_func=module_call_func, obj_attrs=obj_attrs
+    #     )
+    #     if obj.interface != self.name:
+    #         raise PluginError(
+    #             f"Plugin 'interface' attribute on '{obj.name}' plugin does not "
+    #             f"match the name of its interface as specified by entry_points."
+    #         )
+    #     return obj
+
+    @classmethod
+    def plugin_module_to_obj(cls, name, module, obj_attrs={}):
```

Comment:
One small change. Can we make this _plugin_module_to_obj? I'd prefer that this be "private" (as private as anything in Python) and that it match the Yaml version.

Reply:
Done!